lib/llist.c

```diff
@@ -146,24 +146,26 @@ void *Curl_node_take_elem(struct Curl_llist_node *e)
   DEBUGASSERT(list->_init == LLISTINIT);
   DEBUGASSERT(list->_size);
```

Comment:
this will not work when `list` is NULL.

Reply:
No, but the `DEBUGASSERT(list);` above should catch the NULL list case.